README.md

```diff
@@ -1 +1,15 @@
-# digit_match
\ No newline at end of file
+# Digit Match
+Number of matching digits starting with the least significant digit in two integers.
```

Comment:
This is sort of like the opposite of hamming distance (if the shorter number is zero-padded to the same length), is there any value in mentioning it to them or otherwise describing the problem in terms of that idea?

The students are given a weekend warrior exercise to implement a hamming distance algorithm, so it's not outside of their experience depending on when this gets assigned.

Reply:
That's a great context to mention! So as not to increase the homework by adding required reading, I added a mention at the bottom of readme file as an aside.